zebedee-cms/src/main/java/com/github/onsdigital/zebedee/content/page/PageTypeUpdateHook.java

```diff
@@ -0,0 +1,37 @@
+package com.github.onsdigital.zebedee.content.page;
+
+import com.github.onsdigital.zebedee.content.page.base.Page;
+import com.github.onsdigital.zebedee.content.page.base.PageType;
+
+import java.io.IOException;
+import java.util.Map;
+
+/**
+ * Allows different page types to be attached to different page update hooks.
+ */
+public class PageTypeUpdateHook implements PageUpdateHook<Page> {
+
+    Map<PageType, PageUpdateHook> pageUpdateHooks;
```

Comment:
unless there is a need can this be made private?

Reply:
made private and final 👍